src/coord.jl

```diff
@@ -156,7 +156,7 @@ function apply_coordinate(coord::Cartesian, aess::Vector{Gadfly.Aesthetics},
         for var in coord.xvars
             for aes in aess
                 vals = getfield(aes, var)
-                vals === nothing && continue
+                (vals === nothing || eltype(vals) <: Measure) && continue
```

Comment:
if passing through Measures is unrelated to {h,v}bands, i'd leaving it in a separate commit as you have it now, and squashing all the others.

Reply:
This change was required to setup {h,v}band as a statistic. It allows the use of the `h` and `w` values in the aesthetic.

Reply:
A weakness of this approach is that `Geom.band` currently needs to be coupled with another geometry. Otherwise, the following error will occur.

<img width="639" alt="image" src="https://user-images.githubusercontent.com/24236593/69489034-0a299700-0ec7-11ea-9e69-9f52f601bd20.png">

Reply:
Actually, that's easily fixed with https://github.com/GiovineItalia/Gadfly.jl/pull/1264/commits/36f217f7e3cd71fe9e56ffb669fa63627692debc.